docs/charter.md

```diff
@@ -0,0 +1,92 @@
+- provides necessary technologies to govern and secure IoT devices.
+
+### 2.1 Success Criteria
+
+First of all, KubeEdge is designed to address IoT/Edge workload scenarios. We'd like to enable customers manage node & device resources from cloud. With single pane of glass, IT administrators or users can fulfill the needs the same way as resources in data center.
+
+Secondly, KubeEdge is an open architecture based on Kubernetes. We'd like folks from industry, academy, community etc. to contribute and make innovation.
+
+Thirdly, KubeEdge is currently considered as one of the reference architecture. It is our goal to make the cloud/edge communication channel and device management APIs be the standard and acquire more companies to adopt.
+
+### 2.2 Out of Scope
+
+We will not try to go deeper into general IoT and Edge computing discussions as there are excellent papers that already cover these topics in details and we are a open source software community.
+
+## 3. Roles and Organization Management
+
+### Roles
+
+#### Maintainers
+
+  - Run operations and processes governing the SIG
+  - Number: 2-3
+  - Membership tracked in [MAINTAINERS](https://github.com/kubeedge/kubeedge/blob/master/MAINTAINERS)
+
+#### Member
+
+- *SHOULD* maintain health of the community
+- *SHOULD* show sustained contributions to at least one project or to the community
+- *SHOULD* hold some documented role or responsibility in the community and / or at least one project 
+- *MAY* build new functionality for projects
+- *MAY* participate in decision making for the projects they hold roles in
+
+#### Security Contact
+
+- *MUST* be a contact point for the Product Security Committee to reach out to for triaging and handling of incoming issues
+- *MUST* accept the [Embargo Policy](https://git.k8s.io/security/private-distributors-list.md#embargo-policy)
+- Defined in `SECURITY_CONTACTS` files, this is only relevant to the root file in the repository. Template [SECURITY_CONTACTS](https://github.com/kubernetes/kubernetes-template-project/blob/master/SECURITY_CONTACTS)
+
+### Organizational Management
+
+- Community members meets bi-weekly on zoom with agenda in meeting notes
+  - *SHOULD* be facilitated by chairs unless delegated to specific Members
+- Project overview and deep-dive sessions organized for KubeCon/CloudNativeCon
+  - *SHOULD* be organized by chairs unless delegated to specific Members
+- Contributing instructions defined in the CONTRIBUTING.md
+
+#### Project Management
+In addition, community have the following responsibilities to PM:
+
+* identify community annual roadmap
+* identify all community features in the current release
+* actively track / maintain SIG features within docs/proposals
+
+#### Technical processes
+
+Projects of the community MUST use the following processes unless explicitly following alternatives they have defined.
+
+* Proposing and making decisions
+
+   - Proposals sent as KEP PRs and published to googlegroup as announcement
```

Comment:
What is KEP here ? The idea of KEP is from Kubernetes. Does this KEP stand for Kubernetes Enhancement Proposal ?
We don't have an enhancements repo yet.